catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -787,6 +830,18 @@ private void setSchedulerStateForAllLooks(@ThreadScheduler.SchedulerState int st
 		}
 	}
 
+	public void setSpriteToFocusOn(Sprite spriteToFocusOn) {
```

Comment:
You won't need the setters if you extract your changes into a seperate kotlin class.